pkgs/development/tools/rust/cargo-geiger/default.nix

```diff
@@ -0,0 +1,30 @@
+{ stdenv, lib, fetchFromGitHub
+, rustPlatform, pkgconfig
+, openssl, Security }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "cargo-geiger";
+  version = "0.7.3";
+
+  src = fetchFromGitHub {
+    owner = "anderejd";
+    repo = pname;
+    rev = "${pname}-${version}";
+    sha256 = "1lm8dx19svdpg99zbpfcm1272n18y63sq756hf6k99zi51av17xc";
+  };
+
+  doCheck = false;
```

Comment:
why exclude tests?

Reply:
Network access.
```
---- test_package::case_3 stdout ----
-------------- TEST START --------------
━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ Snapshot Differences ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
Snapshot file: cargo-geiger/tests/snapshots/mod__test3_package_with_nested_deps.stderr.snap
Snapshot: test3_package_with_nested_deps.stderr
Source: cargo-geiger/tests/mod.rs:44
Old: n.a.
New: Sun,  3 Nov 2019 00:26:17 +0000

+new results
────────────────────────────────────────────────────────────────────────────────
stderr
────────────┬───────────────────────────────────────────────────────────────────
          1 │+error: failed to load source for a dependency on `itertools`
          2 │+
          3 │+Caused by:
          4 │+  Unable to update https://github.com/rust-itertools/itertools.git?rev=8761fbefb3b209
          5 │+
          6 │+Caused by:
          7 │+  failed to clone into: /build/git/db/itertools-eb89fd56766bcf43
          8 │+
          9 │+Caused by:
         10 │+  failed to resolve address for github.com: Name or service not known; class=Net (12)
         11 │+
────────────┴───────────────────────────────────────────────────────────────────
```

Oddly enough, tests seemed to fail even outside of `nix-build` for the release I selected here, but passed on master.

Regardless, it looks like they won't work without network connectivity.

Reply:
is there a way to selectively disable certain tests?

Reply:
Eh I’d rather not selectively disable tests, but I do want to look at [cargo insta](https://docs.rs/insta/0.12.0/insta/) which this crate seems to use. 

Going to see if there’s anything I can do in `preCheck` or etc.

Reply:
In the case that doesn't work, then yes, [skipping specific tests](https://github.com/NixOS/nixpkgs/blob/63746f6bcf62648445674108c828e7e0c47d8c99/pkgs/development/tools/rust/racer/default.nix#L29) seems pretty straight-forward to me.

Reply:
Decided to settle on disabling a few tests for now.
I was able to compile after skipping 3 out of 7 tests.

I also added a note on how we might be able to run these in the future.